neo/Persistence/LevelDB/Prefixes.cs

```diff
@@ -22,10 +24,11 @@ internal static class Prefixes
         public const byte IX_CurrentStateRoot = 0xc2;
 
         public const byte SYS_Version = 0xf0;
-
         /* Prefixes 0xf1 to 0xff are reserved for external use.
          *
          * Note: The saved consensus state uses the Prefix 0xf4
          */
+
+        public static readonly byte[] ROOT_KEY = Blockchain.GenesisBlock.Hash.ToArray();
```

Comment:
Why not a short value, like `0xf1`?

Reply:
It could be a CRC of this value, but yes, it could be a short one

Reply:
The network magic?